packages/mobile/src/web3/saga.ts

```diff
@@ -221,7 +221,7 @@ export function* assignAccountFromPrivateKey(privateKey: string, mnemonic: strin
     } catch (e) {
       if (
         e === RpcWalletErrors.AccountAlreadyExists ||
-        e === ErrorMessages.GETH_ACCOUNT_ALREADY_EXISTS
+        e?.message === ErrorMessages.GETH_ACCOUNT_ALREADY_EXISTS
```

Comment:
Why is this line the `message` property but the above one the full error object?

Reply:
Yeah, good point, the previous line probably isn't working either, do you know how to trigger that error to test? They both look the same, so if this one is triggering the other one might never do so?

Reply:
I'm not sure how to trigger it. When I added that condition to fix the bug, I just assumed it needed to be in the same format as the condition that was already there.

I agree with you though - I think they both should be checking against the `message` property not the error object. Good catch!